exporter/azure_monitor/azure_monitor.go

```diff
@@ -28,21 +27,20 @@ func NewAzureTraceExporter(o common.Options) (*AzureTraceExporter, error) {
 
 var _ trace.Exporter = (*AzureTraceExporter)(nil)
 
-//Export SpanData to Azure Monitor
+// Export SpanData to Azure Monitor
 // The () before the function name means it is a function of AzureTraceExporter
 func (e *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {
 	baseObj := common.BaseObject {IKey : e.options.InstrumentationKey}
 	envelope := common.Envelope {
 		BaseObject: baseObj,
-		//StartTime:  		sd.StartTime,
 	}
 	envelope.Name = "Microsoft.ApplicationInsights.RemoteDependency"
 
 	transporter := common.Transporter{ 
-		Envel: envelope,
+		Envel: envelope, // Transporter will have more later

```

Comment:
what is `Envel` (why not `Envelope` or something people can understand).

Reply:
I need to capitalize the attribute name to use it publicly. I don't want both the attribute name and the data type to be the exact same. Maybe a different name would help. What do you think of EnvelopeData?